test/e2e/rte.go

```diff
@@ -81,4 +89,32 @@ var _ = ginkgo.Describe("[RTE] Node topology updater", func() {
 			}, time.Minute, 5*time.Second).Should(gomega.BeTrue())
 		})
 	})
+
+	ginkgo.AfterEach(func() {
```

Comment:
so this will run after each test, and now it's fine, but is going to be fine in the future? I'd just move this cleanup inside the tests which create configmaps.

Reply:
Basically, for every test, I also create the ConfigMap in the ```BeforeEach()``` as an empty one, then it's possible to update its content inside the tests themselves or inside inner ```BeforeEach()```

Reply:
If we keep the code symmetric, it's fine to keep the code in `BeforeEach` and `AfterEach`. It's also fine to do a garbage-collection-like thingy and _conditionally_ clean up in `AfterEach`. In general I think it's little worth to create a generic configmap and update it on each test - just easier and simpler to just create the new configmap straight in the test.
last but not least, the default kube framework creates a temp test namespace for each run, so cleanup is done automatically when the suite ends. Sometimes it's worth to clean up explicitely to avoid to pollute the global state, though.

Reply:
BTW  call the ```f.ClientSet``` from the ```AfterEach()``` scope is not valid (end up with nil pointer) I guess that's because the  framework pointer (```f := framework.NewDefaultFramework("rte")```) does not exist in the ```AfterEach()``` context anymore

Reply:
Actually, after a closer look, I see we can't create the ConfigMap during the test at all.
The ConfigMap should be created on the cluster **before**  the rte container is running (the RTE container consumes the ConfigMap). this is not the case because the RTE container is up and running, before the test even start (it was created by ```kubectl create -f ``` in previous steps of the CI.
Need to find a better way to test it.